readme.md

```diff
@@ -17,24 +17,33 @@ $ npm install --save skin-tone
 ```js
 const skinTone = require('skin-tone');
 
-skinTone('👍', 4);
+skinTone('👍', skinTone.BROWN); // or skinTone('👍', 4);
 //=> '👍🏾'
 
-skinTone('👍', 1);
+skinTone('👍', skinTone.WHITE); // or skinTone('👍', 1);
 //=> '👍🏻'
 
 // can also remove skin tone
-skinTone('👍🏾', 0);
+skinTone('👍🏾', skinTone.NONE); // or skinTone('👍🏾', 0);
 //=> '👍'
 
 // just passes it through when not supported
-skinTone('🦄', 5);
+skinTone('🦄', skinTone.DARK_BROWN); // or skinTone('🦄', 5);
```

Comment:
I was thinking of changing it to

``` js 
skinTone('👍', skinTone.BROWN);
skinTone('👍', 4); // or without using the constant
//=> '👍🏾'

skinTone('👍', skinTone.WHITE);
skinTone('👍', 1);
//=> '👍🏻'

...
```

So adding a comment just for the first one and leaving both forms for the rest.